lib/rawQueries.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 
 const Queries = {
+    getPipelineUsageCountQuery: (tablePrefix = '') => `SELECT COUNT(DISTINCT pipelineId) as count, templateId
+        FROM "${tablePrefix}jobs"
+        WHERE templateId IN (:templateIds)
+        GROUP BY templateId;`,
```

Comment:
This would throw syntax error against Postgres and MySql. We need to have 2 separate queries
```suggestion
    getPipelineUsageCountForJobTemplatesQuery: (tablePrefix = '') => `SELECT COUNT(DISTINCT "pipelineId") as count, "templateId"
        FROM "${tablePrefix}jobs"
        WHERE "templateId" IN (:templateIds)
        GROUP BY "templateId"`,
    
    getPipelineUsageCountForJobTemplatesQueryMysql: (tablePrefix = '') => `SELECT COUNT(DISTINCT pipelineId) as count, templateId
        FROM \`${tablePrefix}jobs\`
        WHERE templateId IN (:templateIds)
        GROUP BY templateId`,
```